test/compile/legend.test.ts

```diff
@@ -36,6 +72,14 @@ describe('Legend', function() {
   });
 
   describe('properties.symbols', function() {
-    // FIXME test
+    it('should initialize if filled', function() {
+      const symbol = legend.properties.symbols({field: 'a'}, {}, parseUnitModel({
+          mark: "point",
```

Comment:
Why this legend.properties has an error message of `Property 'properties' does not exist on type 'typeof "/home/zoe/Documents/vl2/vl/vega-lite/src/compile/legend"'.`?